src/components/v5/frame/ColonyHome/partials/TotalInOutBalance/partials/TitledSection.tsx

```diff
@@ -45,9 +51,26 @@ export const TitledSection: FC<TitledSectionProps> = ({
             <div className="mt-0.5 text-md font-medium">{currency}</div>
           </LoadingSkeleton>
         </div>
-        <LoadingSkeleton isLoading={isLoading} className="h-4 w-[70px] rounded">
-          <div className="mt-1 text-xs uppercase text-gray-400">{caption}</div>
-        </LoadingSkeleton>
+        <div className="mt-1 flex flex-row items-center gap-4">
+          <LoadingSkeleton
+            isLoading={isLoading}
+            className="h-4 w-[70px] rounded"
+          >
+            <div className="text-xs uppercase text-gray-400">{caption}</div>
+          </LoadingSkeleton>
+          <LoadingSkeleton
+            isLoading={isLoading}
+            className="h-4 w-[45px] rounded"
+          >
+            <div className="flex flex-row items-center gap-0.5 text-xs font-medium text-blue-400">
+              <ArrowUp
+                size={10}
+                transform={`rotate(${trend.isIncrease ? '0' : '180'})`}
+              />
```

Comment:
In `FundsCardsTotalDescription.tsx` we have another logic for arrow up/down. Don't you want to move that arrow in a separate component, so we can reuse it in FundsCardsTotalDescription and here?